otherlibs/win32unix/unix.ml

```diff
@@ -361,6 +361,7 @@ external isatty : file_descr -> bool = "unix_isatty"
 
 (* Operations on file names *)
 
+(* Unlink is redefined after rmdir definition. *)
```

Comment:
I'd just move the `external` down to above the actual function, then the comment isn't necessary. The precise section containing `unlink` doesn't matter much.